plugins/inputs/http_response/http_response_test.go

```diff
@@ -210,6 +212,69 @@ func TestFields(t *testing.T) {
 	checkOutput(t, &acc, expectedFields, expectedTags, absentFields, nil)
 }
 
+func findInterface() (net.Interface, error) {
+	potential, _ := net.Interfaces()
+
+	for _, i := range potential {
+		// ignore interfaces which are down or not used for broadcast
+		if (i.Flags&net.FlagUp == 0) || (i.Flags&net.FlagBroadcast == 0) {
+			continue
+		}
+
+		if addrs, _ := i.Addrs(); len(addrs) > 0 {
+			// return interface if it has at least one unicast address
+			return i, nil
+		}
+	}
+
+	return net.Interface{}, errors.New("cannot find suitable interface")
+}
+
+func TestInterface(t *testing.T) {
+	var (
+		mux = setUpTestMux()
+		ts  = httptest.NewServer(mux)
+	)
+
+	defer ts.Close()
+
+	intf, err := findInterface()
+	if err != nil {
+		t.Fatal(err)
```

Comment:
I likely would have just `t.Skip(err)`, but then again I believe `httptest.NewServer` requires at least the loopback interface, so this is fine.

Reply:
I can change that if skip is preferred. I didn’t realize skip existed in that form.

Reply:
I can change that if skip is preferred. I didn’t realize skip existed in that form.